app/src/main/java/com/google/jetpackcamera/ui/Routes.kt

```diff
@@ -13,10 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package com.google.jetpackcamera.ui
 
 object Routes {
-    const val PreviewRoute = "preview"
-    const val SettingsRoute = "settings"
-}
\ No newline at end of file
+    const val PREVIEW_ROUTE = "preview"
```

Comment:
I think you need to change where these are used too

Reply:
Yeah apparently `spotlessApply` renamed this automatically but didn't update references. I've updated the references now.